qpc-tools.spec

```diff
@@ -66,5 +66,12 @@ install -D -p -m 644 docs/qpc-tools.1 %{buildroot}%{_mandir}/man1/qpc-tools.1
 %{_mandir}/man1/%{src_name}.1.gz
 
 %changelog
-* Thu Jun 27 2019 Cody Myers <cmyers@redhat.com> 0.1.2
-- Creating the qpc-tools as a spec
+* Thu Oct 24 2019 Cody Myers <cmyers@redhat.com> 0.2.0
+- transitioned to a python client
+- added for support RHEL8
+- added support for Podman on RHEL8 & RHEL/Centos7
+- converted the installation script into an RPM
+- renamed the installer from quipucords-installer to qpc-tools
+- added a role for Redhat Registry authentication
+- added prompts for passwords & Redhat Registry username
+- handled advanced arguements for ansible-extras
```

Comment:
Can we break out some credit for this changelog?

Reply:
change `arguements` to `arguments`